### ISSUE TITLE
Error: Unable to locate package lib32ncurses6 on Ubuntu 18.04.3 LTS

### ISSUE THREAD

**Issue by melashkov** [crazy-max/csgo-server-launcher#67]
#### Steps to reproduce this issue

1. Create brand new image of Ubuntu 18.04.3 LTS
2. Login as root
3. root@CS-GO:~# sudo curl -sSLk https://github.com/crazy-max/csgo-server-launcher/releases/download/v1.14.2/install.sh | sudo bash

#### Expected Behaviour

> Starting CSGO Server Launcher install (v1.14.2)...
>
>Adding i386 architecture...
>Installing required packages...
>Downloading CSGO Server Launcher script...
>Chmod script...
>Install System-V style init script link...
>Downloading CSGO Server Launcher configuration...
>Checking steam user exists...
>Adding steam user...
>Creating /var/steamcmd folder...
>Updating USER in config file...
>Updating IP in config file...
>Updating DIR_STEAMCMD in config file...
>
>Done!
>
>DO NOT FORGET to edit the configuration in '/etc/csgo-server-launcher/csgo-server-launcher.conf'
>Then type:
>  '/etc/init.d/csgo-server-launcher create' to install steam and csgo
>  '/etc/init.d/csgo-server-launcher start' to start the csgo server!

#### Actual Behaviour

> Starting CSGO Server Launcher install (v1.14.2)...
>
>Adding i386 architecture...
>Installing required packages...
>E: Unable to locate package lib32ncurses6
>ERROR: Cannot install required packages...

### Configuration

**Operating system and platform (ex. Debian 9 64bits)** : 18.04.3 (LTS) x64 (Digital Ocean Image)

**csgo-server-launcher version (v1.14.2)** :

### Logs

> Attach these logs to this issue :

> /var/steamcmd/games/csgo/screenlog.*
> root@CS-GO:~# cat /var/steamcmd/games/csgo/screenlog.*
>cat: '/var/steamcmd/games/csgo/screenlog.*': No such file or directory

> $HOME/Steam/logs/stderr.txt
> No such file or directory

### Screenshots

> Sometimes screenshots are better ;)
```sh
You are required to change your password immediately (root enforced)
Welcome to Ubuntu 18.04.3 LTS (GNU/Linux 4.15.0-66-generic x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage

  System information as of Thu Apr  2 09:37:52 UTC 2020

  System load:  0.09              Processes:           104
  Usage of /:   1.6% of 57.98GB   Users logged in:     0
  Memory usage: 12%               IP address for eth0: 165.22.116.10
  Swap usage:   0%

0 packages can be updated.
0 updates are security updates.



The programs included with the Ubuntu system are free software;
the exact distribution terms for each program are described in the
individual files in /usr/share/doc/*/copyright.

Ubuntu comes with ABSOLUTELY NO WARRANTY, to the extent permitted by
applicable law.

Changing password for root.
(current) UNIX password:
Enter new UNIX password:
Retype new UNIX password:
root@CS-GO:~# sudo curl -sSLk https://github.com/crazy-max/csgo-server-launcher/ releases/download/v1.14.2/install.sh | sudo bash

Starting CSGO Server Launcher install (v1.14.2)...

Adding i386 architecture...
Installing required packages...
E: Unable to locate package lib32ncurses6
ERROR: Cannot install required packages...
root@CS-GO:~#
```

PS// if you don't have Digital Ocean Account click [here ](https://m.do.co/c/f7b5d5bec973)to get 100$ credit

**Comment by melashkov** [comment]
Also this command returns 
```sh
root@CS-GO:~# echo $(sed 's/\..*//' /etc/debian_version)
buster/sid
```

**Comment by crazy-max** [comment]
> PS// if you don't have Digital Ocean Account click here to get 100$ credit

This repository is not a self-promoting platform

**Comment by melashkov** [comment]
Sorry about that, do you still want the report? i have found the fix?

**Comment by crazy-max** [comment]
@melashkov See https://github.com/crazy-max/csgo-server-launcher/issues/60#issuecomment-544237899

**Comment by melashkov** [comment]
can you please elaborate on that?
Error two or more packages specified (lib32ncurses6 cosmic)
not sure what it is mean?

I did this change and it fixed issue for me https://github.com/crazy-max/csgo-server-launcher/pull/68

**Comment by crazy-max** [comment]
@melashkov 

> I did this change and it fixed issue for me #68

Yes that should be a good fix.

**Comment by melashkov** [comment]
> @melashkov
> 
> > I did this change and it fixed issue for me #68
> 
> Yes that should be a good fix.

great, happy to help
